openapi/p4pa-debt-position.openapi.yaml

```diff
@@ -108,6 +108,49 @@ paths:
           description: Debt position already exists
         "500":
           description: Internal server error
+  /receipts/{receiptId}:
+    get:
+      tags:
+        - receipt
+      operationId: getReceiptDetail
+      parameters:
+        - name: receiptId
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int64
+          description: The ID of the receipt
+        - name: orgFiscalCode
+          in: query
+          required: true
+          schema:
+            type: string
```

Comment:
Why should you filter also by orgFiscalCode?
Let's keep clean this API, if you you should authorize the invocation, let's do it on BFF instead.
(is the use of orgFiscalCode to make it dirty, it could be more generic if you used organizationId instead)